docs/manual/common/welcome/WhereToStart.md

```diff
@@ -0,0 +1,14 @@
+# Where to begin
```

Comment:
It is because it is too confusing to have Where to Start, Getting Started, etc. In "Where to begin" I wanted to offer the reader of who landed in the Welcome topics the ability to learn by doing -- follow the Getting Started or learn by reading some concepts first. And, as to order, and a topic having the same name as a section, the whole core concepts section will be overhauled. So, I think your concerns will be addressed then.